system/mount.py

```diff
@@ -124,6 +126,8 @@ def set_mount(module, **kwargs):
     origname =  args['name']
     # replace any space in mount name with '\040' to make it fstab compatible (man fstab)
     args['name'] = args['name'].replace(' ', r'\040')
+    # remove trailing slash in mount name (if exists)
+    args['name'] = re.sub('/$','',args['name'])
```

Comment:
Why not use rstrip instead which will remove the need of re module:
`args['name'] = args['name']).rstrip('/')`